appveyor.yml

```diff
@@ -3,11 +3,6 @@ image: Visual Studio 2017
 clone_depth: 10
 skip_tags: true
 
-branches:
```

Comment:
@TylerLeonhardt do you think `master` and `legacy/1.x` is what we might want here?

Reply:
I suspect we were misusing this now that I'm reading up on it. Looks like it should be fine to take out